pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "potodo"
+description = "List .po files to be translated."
+authors = [
+    { name = "Jules Lasne", email = "jules.lasne@gmail.com" },
```

Comment:
Do you want to add yourself @JulienPalard ?

Reply:
I may do, I may not, depends mainly on the wind direction ;)